src/makeCalculator.js

```diff
@@ -3,8 +3,52 @@
 /**
  * @return {object}
  */
+
 function makeCalculator() {
-  // write code here
+  const mathOperate = {
```

Comment:
[NAMING] - The function is named `makeCalculator`, so it should return an object named `calculator` instead of `mathOperate` to align with the naming convention.